docs/quarantine.md

```diff
@@ -102,3 +102,30 @@ After merging this PR the test will be out of quarantine.
 [2]: https://www.thoughtworks.com/en-us/insights/blog/no-more-flaky-tests-go-team
 [3]: https://docs.gitlab.com/ee/development/testing_guide/flaky_tests.html#quarantined-tests
 [on testgrid]: https://testgrid.k8s.io/kubevirt-periodics
+
+# Test Lane Quarantine
+
+There can be cases where required test lanes are flaking in a clustered manner with a large
+number of tests failing each time. This can cause major delays to merging important pull
+requests and overload CI. Individual test case quarantining does not make sense in this case.
+
+A required test lane should be made optional if the following criteria are met:
+* The required test lane has a failure rate higher than 25% in the last 24 hours
+* More than ten individual test cases are causing the required test lane to fail
```

Comment:
Maybe rather use a percentage here - i.e. for test lanes with very small numbers of tests (vgpu) this amount might never be reached.

Reply:
10 failures in the sig-storage lane is 3.24% while 10 failures in the sig-operator lane is 18.2% - due to the wide variety in the number of tests run per lane I'm not sure a percentage would be the best fit here. 

This is just to qualify it as too many tests to quarantine individually.